ote/src/cljs/ote/views/admin/sea_routes.cljs

```diff
@@ -10,59 +10,95 @@
-                                                                  :on-click #(do
-                                                                               (.preventDefault %)
-                                                                               (e! (admin-controller/->ChangeRedirectTo :admin))
-                                                                               (e! (fp/->ChangePage :edit-route {:id id})))}
-                                                              (t-service/localized-text-with-fallback @selected-language name)]]
-                [ui/table-row-column {:style {:width "10%"}} (if published? "Kyllä" "Ei") ]
-                [ui/table-row-column {:style {:width "20%"}}
-                 [:a {:href (str (.-protocol loc) "//" (.-host loc) (.-pathname loc)
-                     "export/gtfs/" (::t-operator/id operator))}
-                  "Lataa gtfs"]]]))]]])]))
+     (when results
+       [:div
+        [:div "Hakuehdoilla löytyi " (count results) " merireittiä."]
+        [ui/table {:selectable false}
+         [ui/table-header {:adjust-for-checkbox false
+                           :display-select-all false}
+          [ui/table-row
+           [ui/table-header-column {:style {:width "25%"}} "Palveluntuottaja"]
+           [ui/table-header-column {:style {:width "25%"}} "Merireitti"]
+           [ui/table-header-column {:style {:width "10%"}} "Julkaistu?"]
+           [ui/table-header-column {:style {:width "15%"}} "Luotu"]
+           [ui/table-header-column {:style {:width "15%"}} "Viim. Ajopäivä"]
+           [ui/table-header-column {:style {:width "10%"}} "GTFS paketti"]]]
+         [ui/table-body {:display-row-checkbox false}
+          (doall
+            (for [{::transit/keys [id name operator-name operator-id published? created] :as sea-route} results
+                  :let [last-active-date (last-date-in-use sea-route)]]
+              ^{:key (str "link_" sea-route)}
+              [ui/table-row {:selectable false}
+               [ui/table-row-column {:style {:width "25%"}} operator-name]
+               [ui/table-row-column {:style {:width "25%"}} [:a {:href (str "/#/edit-route/" id)
+                                                                 :on-click #(do
+                                                                              (.preventDefault %)
+                                                                              (e! (admin-controller/->ChangeRedirectTo :admin))
+                                                                              (e! (fp/->ChangePage :edit-route {:id id})))}
+                                                             (t-service/localized-text-with-fallback @selected-language name)]]
+               [ui/table-row-column {:style {:width "10%"}} (if published? "Kyllä" "Ei")]
+               [ui/table-row-column {:style {:width "15%"}} (time/format-timestamp-for-ui created)]
```

Comment:
Requirement updated, should be "updated"

Reply:
Changed